content/en/providers/getting-started/operations_start_guide/_index.md

```diff
@@ -0,0 +1,208 @@
+Operations, site administrators primarily receive and react on notification of
+one or more incidents at their site. They will also need to react to security
+issues that are at a global level, but affect their site. Site administrators
+should respond to [GGUS tickets](http://ggus.eu) in a suitable time frame and be
+aware of the alarms at their site, eg. through the
+[operations dashboard](https://operations-portal.egi.eu). Sites must only
+operate supported middleware versions. This implies upgrading it from time to
+time. Emergency releases are treated in a special way. See
+[SEC03 EGI-CSIRT Critical Vulnerability Handling](https://wiki.egi.eu/wiki/SEC03_EGI-CSIRT_Critical_Vulnerability_Handling).
+
+All Site management responsibilities are listed in
+[RC OLA document](https://documents.egi.eu/document/31).
+
+#### Site Operations Manager
+
+The person responsible for the site at the political and legal level. S/he is
+responsible for signing the Operations Level Agreement
+([OLA](https://documents.egi.eu/public/ShowDocument?docid=31)) between the Site
+and the NGI that hosts the site operationally. The Site Operations Manager is
+also responsible for assigning and approving the other site roles in the
+[GOCDB](https://goc.egi.eu/). Further, s/he should ensure that administrators
+are subscribed to relevant mailing lists.
+
+#### Site Security Officer
+
+The person responsible for keeping the site compliant with the
+[Security policies](https://wiki.egi.eu/wiki/EGI_CSIRT:Policies). She/he is also
+the primary contact for the NGI Security officer and EGI CSIRT. The Site
+Security Officer deals with security incidents and shall respond to enquiries in
+a timely fashion as defined in the collection of
+[security procedures and policies](https://wiki.egi.eu/wiki/EGI_CSIRT:Policies).
+
+### Regional level
+
+#### Regional Operator on Duty (ROD)
+
+A team responsible for solving problems/incidents in the infrastructure
+according to agreed procedures. ROD (teams) monitor the sites in their region,
+react to problems identified by the monitoring tools, and oversee problems
+through to their resolution. They ensure that problems are properly recorded and
+that the solutions progress according to specified time lines. They also provide
+support to sites and VOs as needed and provide informational flow to oversight
+bodies in cases of non-responsive sites. They ensure that all necessary
+information is available to all parties. The team is provided by each NGI and
+requires procedural knowledge on the process (rather than technical skills) for
+their work. New ROD team members are required to read the
+[ROD Welcome page](https://wiki.egi.eu/wiki/Regional_Operator_on_Duty_welcome)
+and be familiar with
+[ROD wiki page](https://wiki.egi.eu/wiki/Regional_Operator_on_Duty).
+
+#### NGI Security officer
+
+The member of EGI-CSIRT IRTF (Incident Response Task Force) currently on shift.
```

Comment:
Isn't IRTF redundant here?

Reply:
This whole part needs a full refresh, let's review when it's done.